src/Carnac.Logic/IConcurrencyService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Reactive.Concurrency;
+
+namespace Carnac.Logic
+{
+    public interface IConcurrencyService
+    {
+        IScheduler UiScheduler { get; }
```

Comment:
Could we go with `DispatcherScheduler` here (if we're going with idiomatic Rx concepts) or `MainThreadScheduler` here (if we're going with idiomatic ReactiveUI concepts)?